SonarQube.MSBuild.Tasks/SonarQube.MSBuild.Tasks.csproj

```diff
@@ -5,6 +5,7 @@
     <CodeAnalysisRuleSet>SonarQube.MSBuild.Tasks.ruleset</CodeAnalysisRuleSet>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <AssemblyName>SonarQube.Integration.Tasks</AssemblyName>
+    <DebugType>Full</DebugType>
```

Comment:
This is a new-style project so this property needs to be set for code coverage to be collected.
Perhaps we should add it to the build script instead.